src/modals/WalletConnectModal.tsx

```diff
@@ -105,7 +105,7 @@ const WalletConnectModal = ({ onClose, onConnect }: Props) => {
         setErrorState(t`Invalid address group for the WallectConnect connection`)
       }
 
-      return [`${formatChain(chain.networkId, chain.chainGroup)}:${address.publicKey}`]
+      return [`${formatChain(chain.networkId, chain.chainGroup)}:${address.publicKey}/default`]
```

Comment:
Looking at the code that you linked, using a colon `:` as a separator would work as well, correct?

Reply:
I don't think so, using a `:` as the separator causes the error in the screenshot - invalid format

Reply:
ah, I see. ok then!

Reply:
Is there an easy way I can test this so that I can approve the PR? Sorry, my brain is atm fully devoted to the v2.0 of the desktop wallet 🤐